package.json

```diff
@@ -120,7 +120,8 @@
     "@zwave-js/winston-daily-rotate-file": "^4.5.6-1",
     "ansi_up": "^6.0.2",
     "archiver": "^6.0.1",
-    "axios": "^1.5.1",
+    "axios": "^1.6.2",
+    "axios-progress-bar": "^1.2.0",
```

Comment:
I have dropped this dep

Reply:
Oh. Must have been a merge conflict. I'll update.

Reply:
👍🏼